.github/mvn-settings.xml

```diff
@@ -1,30 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<settings>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
     <servers/>
     <profiles>
         <profile>
-            <id>google-mirror-jboss-proxy</id>
+            <id>google-mirror</id>
             <repositories>
                 <repository>
                     <id>google-maven-central</id>
                     <name>GCS Maven Central mirror EU</name>
```

Comment:
I suppose we are not using EU mirror anymore :)

BTW, doesn't maven central choose the right mirror when it recieves the request? Or these are third-party mirrors?

Reply:
We switched to US mirror in main TS too - https://github.com/quarkus-qe/quarkus-test-suite/commit/98a0883ae2445fa7a24433a93a9b3fb542589bde

We started to use EU mirror in early Quarkus days. GH Actions runners are primarily located in USA, Google mirrors are faster than Maven central repos. Quarkus is also using US mirror because of better performance / speed.

Reply:
```suggestion
                    <name>GCS Maven Central mirror</name>
```

Reply:
PR updated